api/payloads/space.go

```diff
@@ -84,3 +85,33 @@ func (l *SpaceList) DecodeFromURLValues(values url.Values) error {
 	l.OrganizationGUIDs = values.Get("organization_guids")
 	return nil
 }
+
+type SpaceDeleteRoutes struct {
+	Unmapped string `json:"unmapped"`
+}
+
+func (d *SpaceDeleteRoutes) SupportedKeys() []string {
+	return []string{"unmapped"}
+}
+
+func (d SpaceDeleteRoutes) Validate() error {
+	return validation.ValidateStruct(&d,
+		validation.Field(&d.Unmapped, validation.Required, validation.By(func(value any) error {
```

Comment:
Can we use `validation.OneOf` here, `true` being the only allowed value?

Reply:
Yes, this is a lot simpler, thanks.

Reply:
@danail-branekov @georgethebeatle What do you think regarding this ?

Reply:
How about 

```
func (d SpaceDeleteRoutes) Validate() error {
	return validation.ValidateStruct(&d,
		validation.Field(&d.Unmapped, validation.Required, validation.StringIn(true, "true").Error("mass delete not supported for mapped routes. Use 'unmapped=true' parameter to delete all unmapped routes")),
	)
}
```

the reason to have the custom `OneOf` validator is to customise the error message in a generic manner. Here we could just not use this generic message and use our own.

Reply:
Or even

```
func (d SpaceDeleteRoutes) Validate() error {
	return validation.ValidateStruct(&d,
		validation.Field(&d.Unmapped,
			validation.Required,
			validation.StringIn(true, "true", "false").Error("must be a boolean"),
			validation.StringIn(true, "true").Error("mass delete not supported for mapped routes. Use 'unmapped=true' parameter to delete all unmapped routes"),
		),
	)
}

```

that validation even makes the tests @klapkov wrote happy :)

Reply:
Yep, this should do the job, thanks.